app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt

```diff
@@ -159,6 +158,11 @@ class EventsFragment : Fragment() {
     private fun showNoInternetScreen(show: Boolean) {
         rootView.homeScreenLL.visibility = if (show) View.VISIBLE else View.GONE
         rootView.noInternetCard.visibility = if (!show) View.VISIBLE else View.GONE
+        rootView.eventsEmptyView.visibility = if (show) View.VISIBLE else View.GONE
+    }
+
+    private fun setEventEmptyText(numOfEvents: Int) {
```

Comment:
All logic belongs to ViewModel

Reply:
can you explain a little bit more because in here, the logic of showing internet connection lost (the code is written by someone before) is also implemented in this file, so should I also try to put that into the ViewModel too?

Reply:
No, just change your code and open an issue to move remaining logic to ViewModel

Reply:
I have changed that in the new pull request #1157 . I will close this pull request because it's really messy, I didn't understand pull request when I made this. Please review # 1157. If you have time, please also take time reviewing #1156 , #1150 , #1148